kyo-bench/src/main/scala/kyo/bench/ForkJoinContentionBench.scala

```diff
@@ -0,0 +1,57 @@
+package kyo.bench
+
+import org.openjdk.jmh.annotations.*
+
+class ForkJoinContentionBench extends Bench.ForkOnly[Unit]:
+
+    val depth     = 1000
+    val parallism = Runtime.getRuntime().availableProcessors()
+    val range     = (0 until depth).toList
+
+    def catsBench() =
+        import cats.*
+        import cats.effect.*
+        import cats.implicits.*
+
+        val forkFiber         = IO.unit.start
+        val forkAllFibers     = Traverse[List].traverse(range)(_ => forkFiber)
+        val forkJoinAllFibers = forkAllFibers.flatMap(fibers => Traverse[List].traverse(fibers)(_.join).void)
+
+        Seq.fill(parallism)(forkJoinAllFibers).parSequence.void
+    end catsBench
+
+    override def kyoBenchFiber() =
+        import kyo.*
+
+        val forkFiber         = Fibers.init(())
+        val forkAllFibers     = Seqs.map(range)(_ => forkFiber)
+        val forkJoinAllFibers = forkAllFibers.flatMap(fibers => Seqs.map(fibers)(_.get).unit)
+
+        Fibers.parallel(Seq.fill(parallism)(forkJoinAllFibers)).unit
+    end kyoBenchFiber
+
+    def zioBench() =
+        import zio.*
+
+        val forkFiber         = ZIO.unit.forkDaemon
```

Comment:
Why forkDaemon and not fork? It shouldn't matter in this context, but I presume you want to avoid overhead of monitoring parent fiber interruption?

Reply:
I'm not sure as well, that came from the original in ZIO: https://github.com/zio/zio/blob/9eb127057495aa36d15c9a3b01f4d2188df9a5ff/benchmarks/src/main/scala/zio/ForkJoinBenchmark.scala#L59